client/layout/masterbar/checkout.jsx

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
Do we need to duplicate `client/layout/masterbar/logged-out.jsx`? For the sake of this test I guess it's ok, but I don't think we need to have a different component for this.